File: content/atlas/source/includes/fts/field-types/string/create_index_example_multi.py
from pymongo import MongoClient
from pymongo.operations import SearchIndexModel

# connect to your Atlas deployment
uri = "<connection-string>"
client = MongoClient(uri)

# set namespace
database = client["sample_mflix"]
collection = database["movies"]

# define your Atlas Search index
search_index_model = SearchIndexModel(
    definition={
        "mappings": {
            "dynamic": False,
            "fields": {
                "title": {
                   "type": "string",
                   "multi": {
                       "english": {
                           "type": "string",
                           "analyzer": "lucene.english"
                       },
                       "french": {
                           "type": "string",
                           "analyzer": "lucene.french"
                       }
                   }
                }
            }
        },
    },
    name="default",
)

# create the index
result = collection.create_search_index(model=search_index_model)
print(f"New index name: {result}")

File: content/atlas/source/includes/fts/field-types/string/create-index-example-multi.js
const { MongoClient } = require("mongodb");

// connect to your Atlas deployment
const uri =  
  "<connection-string>";

const client = new MongoClient(uri);

async function run() {
  try {

    // set namespace
    const database = client.db("sample_mflix");
    const collection = database.collection("movies");

    // define your MongoDB Search index
    const index = {
        name: "default",
        definition: {
            "mappings": {
                "dynamic": false,
                "fields": {
                    "title": {
                        "type": "string",
                        "multi": {
                            "english": {
                                "type": "string",
                                "analyzer": "lucene.english"
                            },
                            "french": {
                                "type": "string",
                                "analyzer": "lucene.french"
                            },
                            "stableSimilarity": {
                                "type": "string",
                                "similarity": { "type": "stableTfl" }
                            }
                        }
                    }
                }
            }
        }
    }

    // run the helper method
    const result = await collection.createSearchIndex(index);
    console.log("New index name: " + result);
  } finally {
    await client.close();
  }
}

run().catch(console.dir);